configs/manager/halfstack.toml

```
@@ -30,6 +30,8 @@ disabled-plugins = []
 
 hide-agents = true
 
+max-session-hangtime = { "PREPARING" = "1h", "TERMINATING" = "30m" }
+
```

Also, name of this config looks a little bit vague; how about renaming it?
```suggestion
session-hang-toleration-threshold = { "PREPARING" = "1h", "TERMINATING" = "30m" }

```